packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -424,6 +426,27 @@ function updateWorkflowNodeState(node: WorkflowNode, state: any) {
 const testNode = (node: WorkflowNode) => {
 	const value = (node.inputs[0].value?.[0] ?? 0) + Math.round(Math.random() * 10);
 	appendOutputPort(node, { type: 'number', label: value.toString(), value });
+
+	console.log('Checking MQ');
+	const auth = useAuthStore();
+	const ttt = new EventSourcePolyfill('/api/simulations/123/partial-result', {
+		headers: {
+			Authorization: `Bearer ${auth.token}`
+		}
+	});
```

Comment:
Please add a new utils in `/api/sse.ts` like we did with `/api/api.ts` to encapsulate the auth for now. So we can handle things like re-auth in the future.

Reply:
Good thinking!